README.md

```diff
@@ -153,8 +155,13 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * What is event loop?
   * What is the difference between call stack and task queue?
 * Explain the differences on the usage of `foo` between `function foo() {}` and `var foo = function() {}`
+* What is object-oriented JavaScript (OOJS)?
```

Comment:
This question is confusing to me as JavaScript is not specifically an object-oriented language. 

> JavaScript, one of the world's most widely distributed languages, has the properties of an untyped functional language, in addition to imperative and object-oriented paradigms.

I'd suggest something more straight forward to test a person's understanding of the language paradigms.

```
Can you explain or illustrate the differences between object-oriented and functional paradigms in JavaScript?
```